cargo/cargo_build_script_runner/bin.rs

```diff
@@ -53,12 +53,28 @@ fn main() {
 
             let mut command = Command::new(exec_root.join(&progname));
             command
-                .args(args)
                 .current_dir(manifest_dir.clone())
                 .env("OUT_DIR", out_dir_abs)
                 .env("CARGO_MANIFEST_DIR", manifest_dir)
                 .env("RUSTC", rustc);
 
+            while let Some(dep_env_path) = args.next() {
+                if let Ok(contents) = read_to_string(dep_env_path) {
```

Comment:
opt: add an error handling here? If the file is unreadable, that's probably a worthy failure mode.